src/modules.js

```diff
@@ -160,7 +160,7 @@ var LibraryManager = {
     }
 
     if (BOOTSTRAPPING_STRUCT_INFO) {
-      libraries = ['library_formatString.js'];
+      libraries = ['library_formatString.js', 'library_stack_trace.js'];
```

Comment:
This change makes it way easier to debug issues with gen_struct_info.. without this calls to aboit() end up with stack overload due to missing stackTrace.